content/projekte/2020-01-21-klimakonzept.md

```diff
@@ -5,6 +5,8 @@ imgname: karlsruhe/klimakonzept.png
 imgcaption: "Foto: Stadt Karlsruhe"
```

Comment:
On https://deploy-preview-229--hungry-heisenberg-4bd67f.netlify.app/projekte/klimakonzept/ I've noticed the image not working. Maybe you could remove the image altogether or fix it in this PR?

Reply:
Thanks for this hint, fixed the image ...